### PR TITLE
fix: Original quality Twitter profile pictures.

### LINKED ISSUE #130: Check if we can get a higher quality thumbnail from Twitter
Current thumbnail is ugly

https://i.nuuls.com/grHZE.png

see if we can find a nicer one

### PR DESCRIPTION
**Pull request by KararTY**
Pull request checklist:

- [x] `CHANGELOG.md` was updated, if applicable

# Description

Note: **I've not tested this fix.**

As per @leon-richardt's link on issue #130, Twitter recommends modifying the received URL to get a higher quality profile picture. This also works for users who haven't set a profile picture.

Closes #130

### CODE REVIEW

**Comment by leon-richardt** [comment]
I think I would prefer doing this in `api.go` where the `TwitterUserApiResponse` is built

**Comment by KararTY** [comment]
> I think I would prefer doing this in `api.go` where the `TwitterUserApiResponse` is built

Good call!

File: internal/resolvers/twitter/api.go [hunk 1]
```diff
@@ -111,5 +112,8 @@ func getUserByName(userName, bearer string) (*TwitterUserApiResponse, error) {
 		return nil, errors.New("unable to process response")
 	}
 
+	// Do modifications to response
```
Comment (leon-richardt) [inline internal/resolvers/twitter/api.go:115]: Either drop this comment or explain why the modification is necessary (could also link to the API documentation here)

**Review by leon-richardt — CHANGES_REQUESTED** [review]

**Comment by zneix** [comment]
For some reason this keeps returning 500

**Review by zneix — APPROVED** [review]
Works as intended :+1: 

sorry leon and karar, my setup was lidl

**Review by pajlada — APPROVED** [review]
Tested and LGTM :+1:

**Pull request merged**